docs/sources/installation/mac.md

```diff
@@ -8,14 +8,13 @@ You can install Docker using Boot2Docker to run `docker` commands at your comman
 Choose this installation if you are familiar with the command-line or plan to
 contribute to the Docker project on GitHub.
 
+[<img src="/installation/images/kitematic.png" alt="Download Kitematic"
```

Comment:
Rather "interesting" way to combine HTML with Markdown; particular reason for not using a plain `<a href=...` here?

No show-stopper, just a bit odd construct